xunit-autolabeler-v2/ast_parser/core/cli.py

```diff
@@ -36,12 +37,196 @@ def _write_output(output: List[str], output_file: str) -> None:
             print(line)
 
 
+def list_region_tags(
+    data_json: str,
+    root_dir: str,
+    show_detected: bool,
+    show_undetected: bool,
+    show_test_counts: bool,
+    show_filenames: bool,
+    output_file: Optional[str] = None
+) -> None:
+    """Lists region tags in a directory.
+
+    This method lists the region tags for snippets parsed from a given
+    directory, filtered based on whether each tag was detected by an AST
+    parser (or not). It can also display test counts and/or source filenames
+    for each region tag.
+
+    Args:
+        data_json: A path to a polyglot_drift_data.json file for the specified
+                   root directory
+        root_dir: A path to the target root directory.
+        show_detected: Whether or not to show region tags that *were* detected
+                       by the AST parser
+        show_undetected: Whether or not to show region tags *not* detected by
+                         the AST parser
+        show_test_counts: Whether or not to show test counts for each
+                          AST-parser-detected region tag
+        show_filenames: Whether or not to show source filenames for each
+                        AST-parser-detected region tag
+        output_file: (Optional) A filepath to write the YAML validation
+                     results to. Results will be written to stdout if this
+                     argument is omitted.
+    """
+    output = []
+
+    if show_undetected and show_test_counts:
+        output.append(
```

Comment:
I'm not sure if it's the right thing to add a warning to the output.

Reply:
Would it be better to send this to `stderr` regardless of `output_file`?